src/component.js

```diff
@@ -37,12 +37,11 @@ const serialize = require('./utilities/serialize');
  *
  * This has a very symmetrical relationship to the userland side of creating a
  * React Component. There are the additional private methods used by other parts
- * of React Core, such as `getPublicInstance` or
- * React <= 15.0 `getNativeNode`
- * React > 15.0 `getHostNode`
+ * of React Core, such as `getPublicInstance` or `getHostNode` (or `getNativeNode`
+ * in React <= 15.0).
```

Comment:
could we leave this one as it was. The list view of the differences between React versions should be really glancable and clear. Less sentence-y; more data-y